Dockerfile

```
@@ -11,7 +11,7 @@ RUN cd /go/src/github.com/mxschmitt/fritzbox_exporter/cmd/exporter && \
 
 FROM alpine
 
-RUN apk add ca-certificates
+RUN apk update && apk add ca-certificates
```

```suggestion
RUN apk add ca-certificates
```
There is no need to to a manual `apk update`. When no apk index exists, there will be automatically an `apk update` done internally before the packages will be installed. In this case ca-certificates.

---

Strange, I wasn't able to build the exporter image without `apk update`. Maybe the auto update is a feature of a newer alpine image and I had an old version because I did not run `docker pull alpine` befor building? In that case maybe the alpine version should be specified in the Dockerfile.

---

Hm okay. Let's add your apk update, which is the way which I saw more often in other Dockerfiles.